Framework/Algorithms/src/AbsorptionCorrection.cpp

```diff
@@ -188,8 +186,9 @@ void AbsorptionCorrection::exec() {
     MantidVec &Y = correctionFactors->dataY(i);
 
     // Loop through the bins in the current spectrum every m_xStep
+    const auto lambdas = m_inputWS->histogram(i).points();
```

Comment:
Note that this clunky interface is temporary: I am expecting changes of the `Histogram` interface, so I did not want to duplicate efforts and provide things like `MatrixWorkspace::points(size_t index)` until the `Histogram` interface has settled. So later this could read
```cpp
const auto lambdas = m_inputWS->points(i);
```